### PR TITLE
add jenkins ci dockerfiles

### PR DESCRIPTION
**Pull request by glorysdj**
this pr is to add dockerfiles for jenkins ci.

---
This change is [Reviewable](https://reviewable.io/reviews/intel-analytics/analytics-zoo/1338)

### CODE REVIEW

File: docker/jenkins-ci/Readme.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+### 1.1 Build master
+First, you should step into the directory jenkins-master and run
+```bash
+sudo docker build --rm -t arda/jenkins-master .
+```
+If proxy is needed for building the image, building arguments related to proxy should be added in the command above. For example,
+```bash
+sudo docker build --build-arg http_proxy=http://yourproxyhost:port \
+                  --build-arg https_proxy=https://yourproxyhost:port \
+                  --rm -t arda/jenkins-master .
```
Comment (jason-dai) [inline docker/jenkins-ci/Readme.md]: don't use arda

File: docker/jenkins-ci/Readme.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Jenkins Docker
+Jenkins is a self-contained, open source automation server which can be used to automate all sorts of tasks related to building, testing, and delivering or deploying software. Here we run Jenkins in Docker, based on Ubuntu 14.04. 
+
+## 1. Build images
+Jenkins has master nodes and slave agents. To make Jenkins well functioning, masters and slaves should be built separatedly. 
+### 1.1 Build master
+First, you should step into the directory jenkins-master and run
+```bash
+sudo docker build --rm -t arda/jenkins-master .
```
Comment (jason-dai) [inline docker/jenkins-ci/Readme.md]: don't use arda here

File: docker/jenkins-ci/Readme.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+There are some extra files other than Dockerfile in the directory jenkins-master.
++ jenkins.sh: launch Jenkins
++ jenkins-support: provide support of upgrading Jenkins and its plugins
++ install-plugins: resolve dependencies and download Jenkins plugins
++ init.groovy: provide initial configuration of launching Jenkins master
+
+### 1.2 Build slaves
+Building Jenkins slave is similar to building Jenkins master. First, step into the directory jenkins-slave and run
+```bash
+sudo docker build --rm -t arda/jenkins-slave .
```
Comment (jason-dai) [inline docker/jenkins-ci/Readme.md]: don't use arda here

File: docker/jenkins-ci/Readme.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+```
+If proxy is needed for building the image, the following command should be run.
+```bash
+sudo docker build --build-arg http_proxy=http://yourproxyhost:port \
+                  --build-arg https_proxy=https://yourproxyhost:port \
+                  --rm -t arda/jenkins-slave .
```
Comment (jason-dai) [inline docker/jenkins-ci/Readme.md]: don't use arda here

**Comment by jason-dai** [comment]
- Change `notebook` folder to `zoo`

- In `jekins-ci`, can we simply provide the dockfile only?

File: docker/jenkins-ci/Readme.md (hunk 1 above)
Comment (glorysdj) [reply docker/jenkins-ci/Readme.md]: ok. will change to intelanalytics

**Comment by glorysdj** [comment]
> * Change `notebook` folder to `zoo`
sure

> * In `jekins-ci`, can we simply provide the dockfile only?
but we need other files to config and start jenkins.

**Pull request merged**